src/main/resources/com/google/api/codegen/ruby/method_sample.snip

```diff
@@ -13,20 +13,32 @@
 @snippet sampleCode_(apiMethod)
   require "{@apiMethod.initCode.apiFileName}"
 
+  {@sampleCodeAliases(apiMethod)}
+
   {@sampleCodeCore(apiMethod)}
 @end
 
-@snippet sampleCodeCore(apiMethod)
+@snippet sampleCodeAliases(apiMethod)
   @join fileImport : apiMethod.initCode.importSection.appImports
     @join alias : fileImport.types
       {@alias.nickname} = {@alias.fullName}
     @end
   @end
-  {@BREAK}
+@end
+
+@snippet sampleCodeCore(apiMethod)
   {@apiMethod.apiVariableName} = {@apiMethod.apiClassName}.new
   @if apiMethod.initCode.lines
     {@initCode(apiMethod.initCode)}
   @end
+  # The different *SampleCode(apiMethod) snippets include calls to
+  # non-declared methods, like:
+  #
+  # @# Process the response.
```

Comment:
I don't think you should put these 'process_xxx' calls here in 'sampleCodeCore' inside method_sample.snip, because these calls are defined just for the purpose of unit tests. SampleGen should stay as the purpose of generating samples. Plus, it will break smoke tests since they don't know what these calls are. So these calls should be moved out of method_sample.snip.